docs/en/reference/shortcuts.md

```diff
@@ -58,6 +58,7 @@ For remembering them easier, here are some thoughts we've put into assigning the
 * `Cmd/Ctrl+B`: Make the selection bold or insert new bold formatting characters.
 * `Cmd/Ctrl+I`: Make the selection italic or insert new italics formatting characters.
 * `Cmd/Ctrl+K`: Insert a new link template or convert the selected text into the caption of a new link. If the clipboard contains a recognisable link, also fill in the braces for the target.
+* `Cmd/Ctrl+T`: Make the current line into an un-checked tasklist item.
```

Comment:
I would suggest to keep it more or less simple, the "un-checked" is more or less self-explanatory, so maybe keep it out?